tests/views/contact.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from __future__ import absolute_import
+
+import pytest
+
+from django.urls import reverse
+
+from pootle_store.models import Unit
+
+
+@pytest.mark.django_db
+def test_contact_report_form_view(client, admin, rf):
+    unit = Unit.objects.last()
+    url = '%s?report=%s' % (reverse('pootle-contact-report-error'), unit.pk)
+    client.login(username=admin.username, password=admin.password)
+    response = client.get(url)
```

Comment:
whenever i see `client.get` or `client.post` im curious as to what the `response.status_code` is